src/main/cljc/vendor/semver/core.cljc

```diff
@@ -0,0 +1,72 @@
+; Courtesy of https://github.com/owainlewis/semver
```

Comment:
This isn't published to Clojars? If not, there aren't any published alternatives up to snuff?

Reply:
An older version is: https://clojars.org/search?q=semver

That it's older doesn't matter, but it has a few calls to `Integer/parseInt` in it, which makes it Java only. The other ones had the same problems AFAICS. I could try submitting a PR to the repo, but he seemed pretty inactive on Github in recent years, so this seemed like the simpler solution for the moment.